### PR TITLE
openresolv: fix openresolv -R command

### LINKED ISSUE #243564: openresolv is missing sed in its wrapper
### Describe the bug

openresolv is missing sed in PATH
I just tried to run that command on unstable
```
nix build .#openresolv
➜ result/bin/resolvconf -R
/nix/store/r0iw20jzng26chwk29pzpmpyrqmdmq2g-openresolv-3.12.0/sbin/.resolvconf-wrapped: line 765: sed: command not found
```

### Expected behavior
according to the help, I was expecting the command:
`-R Show the system service restart command`

### Additional context
was trying to refresh some DNS entry in the cache

### Metadata
nixos-unstable as of today

### PR DESCRIPTION
**Pull request by teto**
which relies on sed

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -11145,7 +11145,9 @@ with pkgs;
 
   openobex = callPackage ../tools/bluetooth/openobex { };
 
-  openresolv = callPackage ../tools/networking/openresolv { };
+  openresolv = callPackage ../tools/networking/openresolv {
+    sed = gnused;
+  };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:11150]:
```suggestion
  openresolv = callPackage ../tools/networking/openresolv { };
```

File: pkgs/tools/networking/openresolv/default.nix [hunk 2]
```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
   installFlags = [ "SYSCONFDIR=$(out)/etc" ];
 
   postInstall = ''
-    wrapProgram "$out/sbin/resolvconf" --set PATH "${coreutils}/bin"
+    wrapProgram "$out/sbin/resolvconf" --set PATH "${lib.makeBinPath [ coreutils sed ]}"
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/openresolv/default.nix:32]:
```suggestion
    wrapProgram "$out/sbin/resolvconf" --set PATH "${lib.makeBinPath [ coreutils gnused ]}"
```

File: pkgs/tools/networking/openresolv/default.nix [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper, coreutils }:
+{ lib, stdenv, fetchFromGitHub, makeWrapper, coreutils, sed }:
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/openresolv/default.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, makeWrapper, coreutils, gnused }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by vcunat** [comment]
🤔  homepage says
> Does not need awk, grep or sed which means we can work without /usr mounted

**Comment by teto** [comment]
the PR fixes an optional non important feature, if homepage mentions not relying on sed as an advantage, let's close this.

**Pull request closed without merging**